whylog/front/whylog_factory.py

```diff
@@ -0,0 +1,32 @@
+import yaml
+
+from whylog.assistant import RegexAssistant
+from whylog.config import YamlConfig
+from whylog.log_reader import LogReader
+from whylog.teacher import Teacher
+
+
+def _teachers_generator(config, pattern_assistant):
+    """
+    Every call of this function return new teacher for new lesson.
+    """
+    while True:
+        yield Teacher({0: ''}, 0, config, pattern_assistant)
+
+
+def whylog_factory(path, pattern_assistant=None):
+    """
+    This function create object of YamlConfig and send this object
```

Comment:
"send" is confusing to me. This function doesn't send YamlConfig anywhere. It uses YamlConfig to create LogReader and teacher_generator.